.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
+node_js:
+- "0.10"
+addons:
+  sauce_connect: true
+env:
+  global:
+  - TRAVIS_LAUNCHES_SAUCE_CONNECT=true
+  - secure: GljUxDK7oCkqvKWzP+OZUdx8lCFJ7FuEqxPH0Cmmijh21KKfvepI2tM6YtQ6dQ3Z8BNWLt5ckvQJNrTm/n49Q+cOJh3ulzesbu0/UdAFRWd0OW0hK9MZGnoTw3DF6NbcYDxeCCiYgq6bIj897RY4fKDszlibvMIS3BgQYMd8DFs=
```

Comment:
You should add comments here indicating what these will be when they are decrypted and which key they are associated with. At some point this file may have many different secure environmental variables.